core/src/main/scala/org/zalando/grafter/GenericReader.scala

```diff
@@ -35,6 +35,14 @@ trait GenericReader {
   implicit def widenReader[R, A, B](r: Reader[R, A])(implicit ev: A <:< B): Reader[R, B] =
     r.map(a => ev(a))
 
+  /**
+   * compose 2 readers instances where their implicit instances are available
+   * This is particularly useful when some components define their configuration in a distinct
+   * library and you want to seamlessly extract that configuration from the application
+   * configuration. See the "In a library" page in the User Guide
+   */
+  def composeReaders[C, T, S](implicit s: Reader[T, S], t: Reader[C, T] ): Reader[C, S] =
```

Comment:
Can we swap the params? It would the easier to read if the param's types matched the result type more closely: `(implicit t: Reader[C, T], s: Reader[T, S] ): Reader[C, S]`